README.md

```diff
@@ -3,58 +3,100 @@
 Hubstats is a rails plugin which allows you to search and monitor pull requests made across a collection of repositories. It also gives extra statistics about users and pull requests not found on GitHub.
 
 ## Setup
+The following setup is designed to only be used when integrating this plugin into a rails application.
 
-The following setup is designed to only be used when integrating this plugin into a rails application or for when adding new migrations.
+ To add a new migration: `rails generate migration <migration-name>`
```

Comment:
I don't see a need for it as its standard Rails stuff.
http://guides.rubyonrails.org/v3.2.21/migrations.html#creating-a-migration